Solutions/ApacheHTTPServer/Package/mainTemplate.json

```diff
@@ -25,464 +25,1554 @@
       "defaultValue": "",
       "type": "string",
       "metadata": {
-        "description": "Workspace name for Log Analytics where Sentinel is setup"
+        "description": "Workspace name for Log Analytics where Microsoft Sentinel is setup"
       }
     },
-    "formattedTimeNow": {
+    "workbook1-name": {
       "type": "string",
-      "defaultValue": "[utcNow('g')]",
+      "defaultValue": "Apache HTTP Server",
+      "minLength": 1,
       "metadata": {
-        "description": "Appended to workbook displayNames to make them unique"
+        "description": "Name for the workbook"
```

Comment:
Suggestion: Can we give a better name to `description` field here instead of `Name for the workbook`?